components/task/details/Details.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Task } from '../../../models/task/task.types';
+import Gallery from './gallery/Gallery';
+import Solutions from './solutions/Solutions';
+import Description from './description/Description';
+
+interface TaskDetailsProps {
+  solutions?: string[];
+  tasks?: Task[];
+  login?: string;
+  photo?: string;
+  images?: string[];
+  setisModalOpen: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+const TaskDetails: React.FC<TaskDetailsProps> = ({
+  solutions,
+  tasks,
+  login,
+  photo,
+  images,
+  setisModalOpen,
+}) => {
+  return (
+    <>
```

Comment:
potrzebny ten fragment?

Reply:
wywaliłem to stąd i ogólnie ten modal przerzuciłem od `Solutions`